dragonfly/block/quartz_ore.go

```diff
@@ -0,0 +1,35 @@
+package block
+
+import "github.com/df-mc/dragonfly/dragonfly/item"
+
+// Nether quartz ore is ore found in the Nether.
+type QuartzOre struct {
```

Comment:
Might be better to call this NetherQuartzOre.

Reply:
Done